tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py27-django{18,111}
+envlist = py27-django{18,111}, py36-django20
```

Comment:
Maybe switch the addition to `py36-django{111,20,21}`?  We'll be on Python 3 with Django 1.11 before we upgrade to an LTS Django 2.x, and Django 2.1 is out already.

Reply:
Agreed - done.